ui/raidboss/data/04-sb/raid/o4s.js

```diff
@@ -352,27 +359,31 @@ export default {
         goOutside: {
           en: 'Go Outside',
           de: 'Nach Außen',
+          fr: 'Allez à l\'extérieur',
           ja: '外に出る',
           cn: '去外面',
           ko: '바깥으로',
         },
         dieOnInside: {
           en: 'Die on Inside',
           de: 'Innen sterben',
+          fr: 'Mourrez à l\'intérieur',
           ja: '中で死ぬ',
           cn: '在里面死亡',
           ko: '안쪽 레이저 맞기',
         },
         dieInInIn: {
           en: 'die in in in',
           de: 'sterben rein rein rein',
+          fr: 'Mourrez--intérieur x3',
```

Comment:
Ce et cette "out out out" ci-dessous sont pour TTS, pas pour l'affichage.

Calling 3 times was just emphasizing that this is very important to get inside.

Reply:
Oh, i didn't notice it was for TTS. Ty, i fix it.

Reply:
You can simply search for "output.dieInInIn", and you would see it was invoked in `tts` function, and wasn't invoked in any `alarmText/alertText/infoText` function.
So it is only for TTS, not for displaying.

Reply:
Ok, thank you.